services/brig/test/integration/API/User/Client.hs

```diff
@@ -946,6 +948,18 @@ testPreKeyRace brig = do
   liftIO $ assertEqual "duplicate prekeys" (length regular) (length (nub regular))
   deleteClient brig uid (clientId c) (Just defPasswordText) !!! const 200 === statusCode
 
+testNewNonce :: Brig -> Http ()
+testNewNonce brig = do
```

Comment:
[optional] if you need more nit-picks, you can compare the two uuids and fail if they are the same.  :)